File: java_backup/my java/anky/Sum1.java
import java.io.*; 
class Sum1 extends Power implements fact
    {
        BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
        int num,sum=0,i=1;
        public int factorial(int p)
        {
            int f=1;
            while(i<=p)
            {
                f=f*i;
                i++;
            }
            return f;
        }
        void disp()throws IOException
        {
            System.out.println("Enter number");
            num=Integer.parseInt(br.readLine());
            for(i=1;i<=num;i++)
            {
                sum=sum+(power(i)/factorial(i));
            }
            System.out.println(sum);
        }
        public static void main()throws IOException
        {
            Sum s=new Sum();
            s.disp();
        }
    }

File: 6th_sem/jimut-matlab/mat/3/sum1.m
%sum=1+(1+2)+(1+2+3)+(1+2+3+..+n) using a function
function s=sum1(n)
    s=0;
    count=0;
        while(1)
            s=s+(n-count);
            count=count+2;
            if((n-count)<0)
                break;
            end
        end 
end  
